packages/contracts/scripts/shared/constants/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './metadata'
+export * from './mnemonic'
+export * from './networks'
```

Comment:
This seems unusual to me that you added this. If it worked before without this, why did you add it?

Reply:
my editor claimed that the import path using `export * from "./constants"` from `packages/contracts/scripts/shared/index.ts` did not point to a corresponding type declaration. Basically, I don't know if not having the index file is correct / valid ts / import resolution or what level the error is occurring, but i have reverted.

Reply:
yeah I agree with @3commascapital 's updates, I guess there were a few refactors.

Reply:
more updates for a future deep linting session